nameserver/server_test.go

```diff
@@ -77,6 +78,15 @@ func TestDNSServer(t *testing.T) {
 	if !(len(r.Answer) > 0) {
 		t.Fatal("Number of answers > 0")
 	}
+	// Now ask a query that we expect to return a lot of data.
+	m.SetQuestion("cisco.com.", dns.TypeANY)
+	m.SetEdns0(UDPBufSize, false) // Special magic to coax the other end into complying
```

Comment:
note that this actually defeats the point of the test, because the dns library will allocate a larger buffer for queries with an EDNS0 record requesting it.